game/scripts/vscripts/modules/illusions/illusions.lua

```diff
@@ -118,7 +140,7 @@ function Illusions:create(info)
 	Illusions:_copySpecialCustomFields(unit, illusion)
 	Illusions:_copyAbilities(unit, illusion)
 	Illusions:_copyItems(unit, illusion)
-	Illusions:_copyAppearance(unit, illusion)
```

Comment:
I think you didn't meant to remove it

Reply:
It causes problems with Terrorblade metamorphosis and is useless with this anyway

Reply:
Indeed, in that case also remove _copyAppereance function itself.

That may cause some issues with https://github.com/ark120202/angel-arena-black-star/blob/02b21a22b55e78ba256f90553f610b782a1333ee/game/scripts/vscripts/heroes/hero_pudge/flesh_heap.lua#L6 though, but that thing should be removed anyway.